src/map/pc.cpp

```diff
@@ -4777,6 +4778,82 @@ bool pc_skill(struct map_session_data* sd, uint16 skill_id, int level, enum e_ad
 	}
 	return true;
 }
+
+bool pc_skill_plagiarism(struct map_session_data *sd, uint16 skill_id, uint16 skill_lv)
+{
+	if (sd != nullptr) {
```

Comment:
```suggestion
	if (sd == nullptr)
		return false;
```
Keep in mind this adjustment will change the tabulation for the whole function. This just removes that extra tab and lets us see the exit sooner!